protocols/interfaces.go

```diff
@@ -101,6 +101,14 @@ type Objective interface {
 	GetStatus() ObjectiveStatus
 }
 
+// VirtualObjective is an Objective that manipulates a ledger channel by exchanging signed proposals.
+type VirtualObjective interface {
```

Comment:
OK I think I understand now. this is the interface satisfied by both `virtualfund` and `virtualdefund`. I think it might be more idiomatic to call this `ProposalReceiver`?

Reply:
`ProposalReceiver` is probably a better name!

Reply:
Done in [17b6ff2](https://github.com/statechannels/go-nitro/pull/864/commits/17b6ff2ea795c587e81307bca80381f290994a3c)